clang/include/clang/CodeGen/ModuleBuilder.h

```diff
@@ -27,6 +27,9 @@ namespace llvm {
   }
 }
 
+// Prefix for __builtin_verbose_trap.
```

Comment:
@ahatanak Does this comment need updating? The prefix isn’t just for __builtin_verbose_trap anymore.